.github/CODEOWNERS

```diff
@@ -13,6 +13,7 @@ available_images.md @aws/dl-containers @aws/dlc-autogluon-reviewers @aws/dlc-neu
 release_images_inference.yml @aws/dl-containers @aws/dlc-autogluon-reviewers @aws/dlc-neuron-reviewers @aws/dlc-eia-reviewers @aws/dlc-trcomp-reviewers @aws/dlc-lmi-reviewers @aws/sagemaker-1p-algorithms @aws/dlc-pytorch-reviewers @aws/dlc-triton-reviewers
 release_images_training.yml @aws/dl-containers @aws/dlc-autogluon-reviewers @aws/dlc-neuron-reviewers @aws/dlc-eia-reviewers @aws/dlc-trcomp-reviewers @aws/sagemaker-1p-algorithms @aws/dlc-pytorch-reviewers
 .release_images_template.yml @aws/dl-containers @aws/dlc-autogluon-reviewers @aws/dlc-neuron-reviewers @aws/dlc-eia-reviewers @aws/dlc-trcomp-reviewers @aws/dlc-pytorch-reviewers @aws/dlc-triton-reviewers
+.data/ignore_ids_safety_scan.json @aws/dl-containers @aws/sagemaker-1p-algorithms
```

Comment:
it appears the file path here is just data/*, not .data - can you update this?

Reply:
Ah thanks for the catch. Updated